src/core/midiPorts.cpp

```diff
@@ -468,4 +468,28 @@ void midiReceive(const MidiMsg& mm, const std::string& recipient)
 }
 
 
+/* -------------------------------------------------------------------------- */
```

Comment:
Very instructive comment. Perhaps you can replace it with what the function does? :-)

Reply:
It's a separator, works quite well then browsing code.

Reply:
I see.